Makefile

```diff
@@ -104,14 +104,13 @@ pluginenv:
 	mkdir -p ${PREFIX}/include/hyprland/wlroots
 	mkdir -p ${PREFIX}/share/pkgconfig
 	
-	find src -name '*.h*' -exec cp --parents '{}' ${PREFIX}/include/hyprland ';'
-	cd subprojects/wlroots/include && find . -name '*.h*' -exec cp --parents '{}' ${PREFIX}/include/hyprland/wlroots ';' && cd ../../..
+	find src -name '*.h*' -print0 | cpio --quiet -0dump ${PREFIX}/include/hyprland
+	cd subprojects/wlroots/include && find . -name '*.h*' -print0 | cpio --quiet -0dump ${PREFIX}/include/hyprland/wlroots && cd ../../..
 	cp ./protocols/*-protocol.h ${PREFIX}/include/hyprland/protocols
 	cp ./build/hyprland.pc ${PREFIX}/share/pkgconfig
-	cp ./build/hyprland.pc /usr/share/pkgconfig
```

Comment:
this is actually quite important because pkg-config will not find the .pc file in `/usr/local/share/pkgconfig`

Reply:
If your pkg-config is broken set `PKG_CONFIG_PATH=/usr/local/share/pkgconfig`.

Reply:
I'm not sure what ArchLinux uses but [pkgconf](https://git.sr.ht/~kaniini/pkgconf) built with `--with-pkg-config-dir=/usr/local/share/pkgconfig` will look there by default.

Reply:
With non-default PREFIX pkg-config may not look there by default. For example, after `make install PREFIX=/tmp/test` pkg-config won't find `hyprland.pc` unless `PKG_CONFIG_PATH=/tmp/test/share/pkgconfig` is set via environ(7).

Reply:
Why not relocate `hyprland.pc` from `share/pkgconfig` to `lib/pkgconfig`? For example, `wayfire.pc` is installed under `lib/pkgconfig`, so it's probably more compatible with various distros.

Reply:
FWIW, Wayfire has header-only API just like Hyprland.

Reply:
sure, as long as it works.

Reply:
Unfortunately, FreeBSD looks under `libdata/pkgconfig` instead of `lib/pkgconfig` by default. Let's keep 2 copies of `hyprland.pc` instead of special-casing by OS.

Reply:
I can't test runtime on Linux, including whether pkg-config still finds `hyprland.pc`.